src/Game.js

```diff
@@ -20,6 +23,37 @@ Game.prototype.roll = function (score) {
   };
 };
 
+Game.prototype.score = function () {
```

Comment:
The score function is pretty big, but you handled the complex logic well. You clearly shown the ability to apply the SRP principle in delegating tasks to other functions